ui/src/app/job-list/job-list.component.html

```diff
@@ -23,10 +23,18 @@ <h2 class="settings-header">Displayed columns</h2>
                       (click)="$event.stopPropagation()">{{ df.display }}</mat-checkbox>
       </mat-list-item>
     </mat-list>
-   <mat-divider></mat-divider>
-   <div class="settings-button-container">
-     <button class="apply-link" (click)="saveSettings()">Done</button>
-   </div>
+    <mat-divider></mat-divider>
+    <div class="field-container">
+      <h2 class="settings-header">Displayed rows/page</h2>
+      <mat-form-field class="page-count">
```

Comment:
Is there a reason not to use the built-in page size picker? https://github.com/DataBiosphere/job-manager/blob/72fd13bfec97ed0abf211e514bc76a803a3527c6/ui/src/app/shared/header/header.component.html#L59

Reply:
Is there a way to use that and allow the user to enter in their own numeric value (and not a dropdown of options)?

Reply:
Not that I'm aware of - here are the docs: https://material.angular.io/components/paginator/api

Is setting an arbitrary value really necessary?